### ISSUE TITLE
multi-language

### ISSUE THREAD

**Issue by sshnevis** [immense/Remotely#639]
hi dears

first i thank you for this useful application. its very wonderful....
I suggest you make the program multilingual.
If you give Excel or a database of words, I will volunteer to translate into Farsi.

**Comment by bloesss** [comment]
And I would translate into german ... at least the End User Interface of the Remote Help

**Comment by sshnevis** [comment]
yes, End User Interface or Get support page is necessary to translate.
its important to make it RTL in arabic and persian pages.

**Comment by JSChasle** [comment]
I would do the french translation on my side

**Comment by cz-vilda** [comment]
I would do the Czech translation

**Comment by flavioxx** [comment]
I also wanted to know which files the texts are in to do the translation.

**Comment by Giuseppe-TD** [comment]
italian too :D i can contribute

**Comment by brendoncarvalho** [comment]
I am willing to translate into Brazilian Portuguese (pt_BR)

**Comment by flavioxx** [comment]
problem is how to translate

> I am willing to translate into Brazilian Portuguese (pt_BR)

**Comment by brendoncarvalho** [comment]
> problem is how to translate
> 
> > I am willing to translate into Brazilian Portuguese (pt_BR)

I recommend including i18n support

Structure suggestion:
```
locales/
  en/
    translation.json
  pt-BR/
    translation.json
```

Sample: `pt-BR/translation.json`
```
{
  "home": "Início",
  "account": "Conta",
  "login": "Entrar",
  "logout": "Sair",
  "about": "Sobre",
  "refresh": "Atualizar",
  "perPage": "por página"
}
```

And that we include the translations via pull request here on github.

**Comment by brendoncarvalho** [comment]
There is even a pull request that tries to implement something similar: https://github.com/immense/Remotely/pull/362

**Comment by saleemshalabi** [comment]
i can help for arabic
